ansible/roles/disable_hyperthreading/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: disable hyper-threading
+  block:
+  - name: install disableHT.sh
```

Comment:
"hyperthread" (or "hyperthreading") isn't that long to type and everything is automated. Can we spell it out in file names and service names so that it's obvious what it does ?
i.e, ``disable-hyperthreading.sh`` or ``disable-hyperthreading.service``